az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/examples/java/WordCount.java

```diff
@@ -46,14 +46,17 @@ public class WordCount extends AbstractHadoopJob {
 
   private final String inputPath;
   private final String outputPath;
-  private boolean forceOutputOverrite;
+  private final boolean forceOutputOverwrite;
+  private final boolean outputDelete;
 
   public WordCount(String name, Props props) {
     super(name, props);
     this.inputPath = props.getString("input.path");
     this.outputPath = props.getString("output.path");
-    this.forceOutputOverrite =
+    this.forceOutputOverwrite =
```

Comment:
It looks like we can set this value `force.output.overwrite` to true and then get the concurrent instances passing! :)

Reply:
It won't work as expected. Thinking about this timeline in order when two WordCount instances run:
1. Instance1 started
2. Instance2 started
3. Instance1 overwrite the output by deleting the HDFS files
4. Instance2 overwrite the output by deleting the HDFS files
5. Both Instance1 and instance2 tries to writing data to same output.path at same timestamp. HDFS will report exception saying: output.path is written by another process and corrupted